build/install.bzl

```diff
@@ -0,0 +1,62 @@
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Installs a file to the actual workspace outside of the sandbox.
+"""
+
+load("@bazel_skylib//lib:paths.bzl", "paths")
+
+def _install_impl(ctx):
+
+    cmd = ""
+
+    for s in ctx.attr.srcs:
+      files = s.files.to_list()
+      file = files[0].path
+      root_file = paths.relativize(file,"bazel-out/k8-fastbuild/genfiles/")
```

Comment:
define a build_dir outside the for loop:

```python
    build_dir = "bazel-out/{}-{}/genfiles".format(ctx.var["TARGET_CPU"], ctx.var["COMPILATION_MODE"])
```

and then this line can be

```python
        root_file = paths.relativize(file, build_dir)
```

Reply:
lol or just `ctx.var["GENDIR"]` thx Bazel

Reply:
Omg.